### ISSUE TITLE
'flex-direction: inherit' transforms to 'vertical/normal' instead of 'inherit' on old box properties

### ISSUE THREAD

**Issue by celebro** [postcss/autoprefixer#915]
```css
/* input: */
flex-direction: inherit;

/* autput: */
-webkit-box-orient: vertical;
-webkit-box-direction: normal;
-webkit-flex-direction: inherit;
-moz-box-orient: vertical;
-moz-box-direction: normal;
-ms-flex-direction: inherit;
flex-direction: inherit
```

Shouldn't all properties have value 'inherit'?

**Comment by ai** [comment]
Yeap, seems like a bug. I will try to fix it on this weekend.

**Comment by ai** [comment]
Done a2ef2a4

**Comment by ai** [comment]
Released in 7.1.5.

**Comment by celebro** [comment]
Thank you, much appreciated!
